simulariumio/data_objects/simularium_file_data.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict, Union
+from .trajectory_data import TrajectoryData
+from .frame_data import FrameData
+
+
+class SimulariumFileData:
```

Comment:
Since this looks like the same API I set up in the viewer, I think it might be important to note that this api is good for reading but useless for writing.   
Why does the constructor take both a file name and a file contents?  Are they mutually exclusive?

Reply:
I don't think the file name is actually necessary. I had thought about removing it before, but evidently I did not. I'm just going to remove that

- [ ] remove file_name param